tests/test_elastic.py

```diff
@@ -438,6 +441,40 @@ def test_search_partial(self):
         assert_equal(len(contribs['users']), 0)
 
 
+    def test_search_fullname_special_character(self):
+        """Verify that searching for a fullname with a special character yields
```

Comment:
Thank you for adding these tests. 

As per https://cosdev.readthedocs.org/en/latest/process/testing.html#general-testing-guidelines, we prefer descriptive test method names to docstrings, in order to make it easier to locate tests when they fail. I will convert these docstrings to code comments when I merge.